docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -67,12 +67,18 @@ To define configurable options for your plugin, describe them in the ``DOCUMENTA
         ini:
           - section: section_of_ansible.cfg_where_this_config_option_is_defined
             key: key_used_in_ansible.cfg
+        vars:
+          - name: name_of_ansible_var
+          - name: name_of_second_var
+            version_added: X.x
         required: True/False
         type: boolean/float/integer/list/none/path/pathlist/pathspec/string/tmppath
         version_added: X.x
 
 To access the configuration settings in your plugin, use ``self.get_option(<option_name>)``. For the plugin types (such as 'become', 'cache', 'callback', 'cliconf', 'connection', 'httpapi', 'inventory', 'lookup', 'netconf', 'shell', and 'vars') that support embedded documentation, the controller pre-populates the settings. If you need to populate settings explicitly, use a ``self.set_options()`` call.
 
+Precedence rules for values in Ansible, including from Configuration sources like this, are available in :ref:`general_precedence_rules`. When there are multiple values from the same category, the last defined wins. In the above configuration block for example, if both ``name_of_ansible_var`` and ``name_of_second_var`` are defined, the value of the ``option_name`` option will be the value of ``name_of_second_var`` (assuming the value did not get assigned from a higher precedence source).
```

Comment:
I agree with @jborean93 (on IRC) that the last part is not really needed (since that's already said in the sentence before). I think omitting it makes it easier to read. @samccann what do you think?
```suggestion
Precedence rules for values in Ansible, including from Configuration sources like this, are available in :ref:`general_precedence_rules`. When there are multiple values from the same category, the last defined wins. In the above configuration block for example, if both ``name_of_ansible_var`` and ``name_of_second_var`` are defined, the value of the ``option_name`` option will be the value of ``name_of_second_var``.
```

Reply:
```suggestion
Configuration sources follow the precedence rules for values in Ansible. 
When there are multiple values from the same category, the value defined last takes precedence. For example, in the above configuration block, if both ``name_of_ansible_var`` and ``name_of_second_var`` are defined, the value of the ``option_name`` option will be the value of ``name_of_second_var`` (assuming the value did not get assigned from a higher precedence source). Refer to :ref:`general_precedence_rules` for further information.
```

Reply:
@felixfontein I agree that omitting the last part is better. I moved the reference to general precedence rules to the end of the sentence.